bower.json

```diff
@@ -3,7 +3,7 @@
   "description": "YouTube video playback web component.",
   "homepage": "https://googlewebcomponents.github.io/google-youtube",
   "main": "google-youtube.html",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
Yeah, I'm not sure what the current release process is, but I would leave the version bump out of this PR while we figure that out.